README.md

```diff
@@ -113,6 +113,7 @@ If you believe you've found an issue with this tool, please [create an issue](ht
 | Chinese Traditional (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)                                 | UI                                   |
 | Czech                                       | cz    | [Matto58](https://github.com/Matto58)                                  | UI                                   |
 | English                                     | en    | -                                                                      | UI and README                        |
+| Finnish (Suomen kieli)                      | fi    | [ZapX5](https://github.com/ZapX5)                                      | UI and [README](./README-Finnish.md) |
```

Comment:
On line 17 we reference `README-Chinese.md` - could you do this also for your `README-Finnish.md`? It will mean others can see it when they first visit the repo.